src/pages/home/report/ReportActionCompose/SuggestionMention.tsx

```diff
@@ -333,11 +346,8 @@ function SuggestionMention(
     }, [selection, calculateMentionSuggestion]);
 
     useEffect(() => {
-        const foundSuggestionsCount = suggestionValues.suggestedMentions.length;
-        if (suggestionValues.prefixType === '#' && foundSuggestionsCount < 5) {
-            ReportUserActions.searchInServer(value, policyID);
-        }
-    }, [suggestionValues.suggestedMentions.length, suggestionValues.prefixType, policyID, value]);
+        debouncedSearchInServer();
```

Comment:
I think this isn't triggering the search when it is supposed to. You can see it in the video that when I type multiple characters that yields only 1 result the search isn't triggered, it is also not triggered afterwards when I start deleting characters.

https://github.com/Expensify/App/assets/5201282/7b1ca7d1-ab49-405d-8575-1b528bc3f240

Reply:
Haven't noticed that :( 

The missing server calls should be fixed now (wrong usage of `useDebounce` hook)